# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: fix collation rules for hy_AM
# DP:   Apply patch#13 from BZ68.
# DP:   Thanks to patches/locale/LC_COLLATE-keywords-ordering.diff,
# DP:   the limitations explained in this bugreport do not apply and
# DP:   collation rules can be defined the right way.
# DP: Related bugs: BZ68
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Pablo Saratxaga, Ani <petrosyan@gmail.com>
# DP: Upstream status: BZ68
# DP: Date: 2006-10-02

Index: glibc-2.5/localedata/locales/hy_AM
===================================================================
--- glibc-2.5.orig/localedata/locales/hy_AM
+++ glibc-2.5/localedata/locales/hy_AM
@@ -68,6 +68,176 @@
 
 LC_COLLATE
 copy "iso14651_t1"
+
+collating-symbol <ARM-AYB>
+collating-symbol <ARM-BEN>
+collating-symbol <ARM-EN>
+collating-symbol <ARM-GIM>
+collating-symbol <ARM-YECH>
+collating-symbol <ARM-DA>
+collating-symbol <ARM-ECH>
+collating-symbol <ARM-ZA>
+collating-symbol <ARM-E>
+collating-symbol <ARM-AT>
+collating-symbol <ARM-TO>
+collating-symbol <ARM-ZHE>
+collating-symbol <ARM-INI>
+collating-symbol <ARM-LYUN>
+collating-symbol <ARM-KHE>
+collating-symbol <ARM-TSA>
+collating-symbol <ARM-KEN>
+collating-symbol <ARM-HO>
+collating-symbol <ARM-DZA>
+collating-symbol <ARM-GHAT>
+collating-symbol <ARM-TCHE>
+collating-symbol <ARM-MEN>
+collating-symbol <ARM-HI>
+collating-symbol <ARM-NU>
+collating-symbol <ARM-SHA>
+collating-symbol <ARM-VO>
+collating-symbol <ARM-CHA>
+collating-symbol <ARM-PE>
+collating-symbol <ARM-JE>
+collating-symbol <ARM-RA>
+collating-symbol <ARM-SE>
+collating-symbol <ARM-VEV>
+collating-symbol <ARM-TYUN>
+collating-symbol <ARM-RE>
+collating-symbol <ARM-TSO>
+collating-symbol <ARM-VYUN>
+collating-symbol <ARM-PYUR>
+collating-symbol <ARM-KE>
+collating-symbol <ARM-O>
+collating-symbol <ARM-FE>
+
+reorder-after <CYR-YA>
+<ARM-AYB>
+<ARM-BEN>
+<ARM-EN>
+<ARM-GIM>
+<ARM-YECH>
+<ARM-DA>
+<ARM-ECH>
+<ARM-ZA>
+<ARM-E>
+<ARM-AT>
+<ARM-TO>
+<ARM-ZHE>
+<ARM-INI>
+<ARM-LYUN>
+<ARM-KHE>
+<ARM-TSA>
+<ARM-KEN>
+<ARM-HO>
+<ARM-DZA>
+<ARM-GHAT>
+<ARM-TCHE>
+<ARM-MEN>
+<ARM-HI>
+<ARM-NU>
+<ARM-SHA>
+<ARM-VO>
+<ARM-CHA>
+<ARM-PE>
+<ARM-JE>
+<ARM-RA>
+<ARM-SE>
+<ARM-VEV>
+<ARM-TYUN>
+<ARM-RE>
+<ARM-TSO>
+<ARM-VYUN>
+<ARM-PYUR>
+<ARM-KE>
+<ARM-O>
+<ARM-FE>
+reorder-end
+
+script <ARMENIAN>
+
+order_start <ARMENIAN>;forward;forward;forward;forward,position
+<U0561> <ARM-AYB>;<BAS>;<MIN>;IGNORE
+<U0562> <ARM-BEN>;<BAS>;<MIN>;IGNORE
+<U0563> <ARM-GIM>;<BAS>;<MIN>;IGNORE
+<U0564> <ARM-DA>;<BAS>;<MIN>;IGNORE
+<U0565> <ARM-YECH>;<BAS>;<MIN>;IGNORE
+<U0566> <ARM-ZA>;<BAS>;<MIN>;IGNORE
+<U0567> <ARM-E>;<BAS>;<MIN>;IGNORE
+<U0568> <ARM-AT>;<BAS>;<MIN>;IGNORE
+<U0569> <ARM-TO>;<BAS>;<MIN>;IGNORE
+<U056A> <ARM-ZHE>;<BAS>;<MIN>;IGNORE
+<U056B> <ARM-INI>;<BAS>;<MIN>;IGNORE
+<U056C> <ARM-LYUN>;<BAS>;<MIN>;IGNORE
+<U056D> <ARM-KHE>;<BAS>;<MIN>;IGNORE
+<U056E> <ARM-TSA>;<BAS>;<MIN>;IGNORE
+<U056F> <ARM-KEN>;<BAS>;<MIN>;IGNORE
+<U0570> <ARM-HO>;<BAS>;<MIN>;IGNORE
+<U0571> <ARM-DZA>;<BAS>;<MIN>;IGNORE
+<U0572> <ARM-GHAT>;<BAS>;<MIN>;IGNORE
+<U0573> <ARM-TCHE>;<BAS>;<MIN>;IGNORE
+<U0574> <ARM-MEN>;<BAS>;<MIN>;IGNORE
+<U0575> <ARM-HI>;<BAS>;<MIN>;IGNORE
+<U0576> <ARM-NU>;<BAS>;<MIN>;IGNORE
+<U0577> <ARM-SHA>;<BAS>;<MIN>;IGNORE
+<U0578> <ARM-VO>;<BAS>;<MIN>;IGNORE
+<U0579> <ARM-CHA>;<BAS>;<MIN>;IGNORE
+<U057A> <ARM-PE>;<BAS>;<MIN>;IGNORE
+<U057B> <ARM-JE>;<BAS>;<MIN>;IGNORE
+<U057C> <ARM-RA>;<BAS>;<MIN>;IGNORE
+<U057D> <ARM-SE>;<BAS>;<MIN>;IGNORE
+<U057E> <ARM-VEV>;<BAS>;<MIN>;IGNORE
+<U057F> <ARM-TYUN>;<BAS>;<MIN>;IGNORE
+<U0580> <ARM-RE>;<BAS>;<MIN>;IGNORE
+<U0581> <ARM-TSO>;<BAS>;<MIN>;IGNORE
+<U0582> <ARM-VYUN>;<BAS>;<MIN>;IGNORE
+<U0583> <ARM-PYUR>;<BAS>;<MIN>;IGNORE
+<U0584> <ARM-KE>;<BAS>;<MIN>;IGNORE
+<U0585> <ARM-O>;<BAS>;<MIN>;IGNORE
+<U0586> <ARM-FE>;<BAS>;<MIN>;IGNORE
+<U0531> <ARM-AYB>;<BAS>;<CAP>;IGNORE
+<U0532> <ARM-BEN>;<BAS>;<CAP>;IGNORE
+<U0533> <ARM-GIM>;<BAS>;<CAP>;IGNORE
+<U0534> <ARM-DA>;<BAS>;<CAP>;IGNORE
+<U0535> <ARM-YECH>;<BAS>;<CAP>;IGNORE
+<U0536> <ARM-ZA>;<BAS>;<CAP>;IGNORE
+<U0537> <ARM-E>;<BAS>;<CAP>;IGNORE
+<U0538> <ARM-AT>;<BAS>;<CAP>;IGNORE
+<U0539> <ARM-TO>;<BAS>;<CAP>;IGNORE
+<U053A> <ARM-ZHE>;<BAS>;<CAP>;IGNORE
+<U053B> <ARM-INI>;<BAS>;<CAP>;IGNORE
+<U053C> <ARM-LYUN>;<BAS>;<CAP>;IGNORE
+<U053D> <ARM-KHE>;<BAS>;<CAP>;IGNORE
+<U053E> <ARM-TSA>;<BAS>;<CAP>;IGNORE
+<U053F> <ARM-KEN>;<BAS>;<CAP>;IGNORE
+<U0540> <ARM-HO>;<BAS>;<CAP>;IGNORE
+<U0541> <ARM-DZA>;<BAS>;<CAP>;IGNORE
+<U0542> <ARM-GHAT>;<BAS>;<CAP>;IGNORE
+<U0543> <ARM-TCHE>;<BAS>;<CAP>;IGNORE
+<U0544> <ARM-MEN>;<BAS>;<CAP>;IGNORE
+<U0545> <ARM-HI>;<BAS>;<CAP>;IGNORE
+<U0546> <ARM-NU>;<BAS>;<CAP>;IGNORE
+<U0547> <ARM-SHA>;<BAS>;<CAP>;IGNORE
+<U0548> <ARM-VO>;<BAS>;<CAP>;IGNORE
+<U0549> <ARM-CHA>;<BAS>;<CAP>;IGNORE
+<U054A> <ARM-PE>;<BAS>;<CAP>;IGNORE
+<U054B> <ARM-JE>;<BAS>;<CAP>;IGNORE
+<U054C> <ARM-RA>;<BAS>;<CAP>;IGNORE
+<U054D> <ARM-SE>;<BAS>;<CAP>;IGNORE
+<U054E> <ARM-VEV>;<BAS>;<CAP>;IGNORE
+<U054F> <ARM-TYUN>;<BAS>;<CAP>;IGNORE
+<U0550> <ARM-RE>;<BAS>;<CAP>;IGNORE
+<U0551> <ARM-TSO>;<BAS>;<CAP>;IGNORE
+<U0552> <ARM-VYUN>;<BAS>;<CAP>;IGNORE
+<U0553> <ARM-PYUR>;<BAS>;<CAP>;IGNORE
+<U0554> <ARM-KE>;<BAS>;<CAP>;IGNORE
+<U0555> <ARM-O>;<BAS>;<CAP>;IGNORE
+<U0556> <ARM-FE>;<BAS>;<CAP>;IGNORE
+order_end
+
+reorder-sections-after <CYRIL>
+<ARMENIAN>
+reorder-sections-end
+
 END LC_COLLATE
 
 LC_CTYPE
